src/osd/scheduler/mClockScheduler.cc

```diff
@@ -401,6 +402,7 @@ void mClockScheduler::dump(ceph::Formatter &f) const
   // Display queue sizes
   f.open_object_section("queue_sizes");
   f.dump_int("immediate", immediate.size());
+  f.dump_int("high_queue", high_queue.size());
```

Comment:
'high_queue' is a terrible name. And it does not fit the others: you do not have 'immediate_queue', for example.
'priority_queue'? 'high_priority'?

Reply:
I don't hate it, but I agree they should follow the same convention whatever is chosen.